### ISSUE TITLE
Just cloned: Runtime Error

### ISSUE THREAD

**Issue by gbonline** [NativeScript/nativescript-sdk-examples-ng#243]
Hi on ubuntu 16.04, for Android, with tns 3.3.1, just downloaded and executed with `tns run android`, then for each elements in list I get Error like this:

```
> JS: ERROR Error: Uncaught (in promise): EmptyError: no elements in sequence
> JS: Error: no elements in sequence
> JS:     at EmptyError.ZoneAwareError (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/nativescript-angular/zone-js/dist/zone-nativescript.js:993:33)
> JS:     at new EmptyError (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/util/EmptyError.js:27:34)
> JS:     at FirstSubscriber._complete (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/operators/first.js:154:31)
> JS:     at FirstSubscriber.Subscriber.complete (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/Subscriber.js:121:18)
> JS:     at MergeMapSubscriber._complete (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/operators/mergeMap.js:144:30)
> JS:     at MergeMapSubscriber.Subscriber.complete (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/Subscriber.js:121:18)
> JS:     at MapSubscriber.Subscriber._complete (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/Subscriber.js:139:26)
> JS:     at MapSubscriber.Subscriber.complete (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/Subscriber.js:121:18)
> JS:     at EmptyObservable._subscribe (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/observable/EmptyObservable.js:82:24)
> JS:     at EmptyObservable.Observable._trySubscribe (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/Observable.js:173:25)
> JS:     at EmptyObservable.Observable.subscribe (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/Observable.js:161:65)
> JS:     at MapOperator.call (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/operators/map.js:62:23)
> JS:     at Observable.subscribe (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/Observable.js:158:22)
> JS:     at MergeMapOperator.call (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/operators/mergeMap.js:93:23)
> JS:     at Observable.subscribe (file:///data/data/org.nativescript.nativescriptsdkexamplesng/files/app/tns_modules/rxjs/Observable.js:158:22)
> 
```

**Comment by NickIliev** [comment]
@gbonline looks like your zone.js version is mismatched. Can you try removing your `node_modules`, `platforms` and `package--lock.json` file and then rebuild the project.

**Comment by SvetoslavTsenov** [comment]
Hey @gbonline, the problem looks to be in rxjs.
This is the issue related to this exception:
https://github.com/angular/angular/issues/20752
Could you please make sure that your rxjs dependency version is less 
than "rxjs": "5.5.3"

**Comment by SivaPrasanna45** [comment]
How can i solve this issue with RxJS5.5.3 itself? @SvetoslavTsenov @NickIliev

**Comment by sis0k0** [comment]
@SivaPrasanna45, you add `pathMatch: 'full'` to all your routes. Check out this comment: https://github.com/ReactiveX/rxjs/issues/3151#issuecomment-348776875.

**Comment by gbonline** [comment]
@SvetoslavTsenov : Yes I have rxjs@5.5.3 ! I will try to downgrade
but thanks to @sis0k0  I tried adding pathMatch='full' to the routes and it works. The are so many routes.....

**Comment by NickIliev** [comment]
@gbonline this issue should now be fixed with `rxjs 5.5.3` (no need to apply `pathmatch: 'full'`)
